backend/packages/Upgrade/test/unit/controllers/StratificationController.test.ts

```diff
@@ -0,0 +1,72 @@
+import app from '../../utils/expressApp';
+import request from 'supertest';
+import fs from 'fs';
+import path from 'path';
+import { Parser } from 'json2csv';
+import { configureLogger } from '../../utils/logger';
+import { useContainer as routingUseContainer } from 'routing-controllers';
+import { Container } from 'typedi';
+import StratificationServiceMock from './mocks/StratificationServiceMock';
+import { StratificationService } from '../../../src/api/services/StratificationService';
+import { useContainer as classValidatorUseContainer } from 'class-validator';
+import { useContainer as ormUseContainer } from 'typeorm';
+
+describe('Stratification Controller Testing', () => {
+  beforeAll(() => {
+    configureLogger();
+    routingUseContainer(Container);
+    ormUseContainer(Container);
+    classValidatorUseContainer(Container);
+
+    Container.set(StratificationService, new StratificationServiceMock());
+  });
+
+  afterAll(() => {
+    Container.reset();
+  });
+
+  test('Get request for /api/stratification', async (done) => {
+    await request(app).get('/api/stratification').expect('Content-Type', /json/).expect(200);
+    done();
+  });
+
+  const factorName = 'factor1';
```

Comment:
These code should be moved inside the testcase where it is used.

Reply:
resolved